bluemix/configuration/core_config/bx_config_test.go

```diff
@@ -454,6 +454,19 @@ func TestMOD(t *testing.T) {
 	t.Cleanup(cleanupConfigFiles)
 }
 
+func TestLastUpdateSessionTime(t *testing.T) {
+
+	config := prepareConfigForCLI(`{}`, t)
+
+	// check initial state
+	assert.Empty(t, config.LastSessionUpdateTime())
+
+	// Set last session update time and check that the timestamp is set
+	config.SetLastSessionUpdateTime()
+
+	assert.NotEmpty(t, config.LastSessionUpdateTime())
```

Comment:
I understand not checking the exact time, but should probably at least validate the time that came back is in the expected format.

Reply:
Fair point. I will make the adjustment